.env.example

```diff
@@ -1,4 +1,4 @@
-API_BASE_URL=http://127.0.0.1:8080
+API_BASE_URL=http://127.0.0.1:8080/
```

Comment:
is this change needed?

Reply:
Yes new version of imposter needed this.